### DISCUSSION TITLE
Settings go blank on Mac

### DISCUSSION THREAD

**Discussion by cansas** [cgiesche/streamdeck-homeassistant#214] [Help and Support]
Not sure how, but the settings are going blank and losing the primary configuration on Mac Mini M1. May have been a reboot which caused but it is consistent. I enter the settings, configure a button but it does not fire and when I go back the button config is blank.

**Comment by Grayda** [comment]
Same thing on Windows 10. I haven't dived into the source to find out why though.

**Comment by t4b4sc0** [comment]
same here, reboot, config gone. are there logs somewhere?

**Comment by jorblad** [comment]
I seem to have the sam thing in a mac, I can configure it and select the entities but i dont get any data from sensors and switches dont trigger in homeassistant.

**Comment by Tritiumnzlol** [comment]
Getting this on Windows 11 too, everytime after a reboot.. It seems strange that the plugin doesn't use the 'accounts' feature to store the secrets, and has to have the same first section on all buttons.

**Comment by cgiesche** [comment]
I can not find any information regarding the "accounts" feature in the stream deck API documentation. It seems like this is not a public feature that every plugin developer can use.

What is really strange: I am storing settings as every other plugin does (via the official stream deck api). There is not much that can go wrong 🤔
